zulipterminal/ui_tools/buttons.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         prefix_character: Union[str, Tuple[Any, str]]='\N{BULLET}',
         text_color: Optional[str]=None,
         count: int=0,
+        count_style: Optional[str]=None,
```

Comment:
This doesn't seem to have been done yet? Did you understand the parts I was referring to?

Reply:
Sorry about that, I pulled apart the commit in a new branch (in case I royally screwed things up) and it seems I accidentally pushed the old branch instead of the new one. The correct version is now up.

I felt it made the most sense to break it into three separate commits:
1. Refactor StarredButton to actually display the count (it uses the existing unread count styling).
2. Add the optional count_style arg to TopButton, refactor the update_count function and add define the style for the various buttons.
3. Add a new style - called starred_count - to themes and change the StarredButton's count_style to it. 

I'd be happy to reorganize or squash two of those together if you think I should. Sorry again about not having the correct version up